pkgs/development/tools/rust/cargo-fund/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, rustPlatform, Security, openssl, pkgconfig, libiconv, curl }:
```

```suggestion
{ stdenv, fetchFromGitHub, pkg-config, rustPlatform, Security, curl, openssl, libiconv }:
```

`pkgconfig` is an alias.

Also (though it is a bit nitpicky), the typical order is `nativeBuildInputs` before `buildInputs`.